README.md

```diff
@@ -304,24 +305,24 @@ Please note that the default for bracket is `square` defined by the [attribute](
 
 #### Lowercase only ASCII characters in auto link ids
 
-The plugin lowercase all alphabets within auto link ids default.
+The plugin lowercase **only ascii alphabets**(`a` to `z`) within auto link ids default.
```

Comment:
Rephrase: "By default the plugin lowercases ASCII based alphabets **only** (`a`to `z`) for auto links"